components/NexBridgeDashboard/EditBridgeModal.tsx

```diff
@@ -63,63 +63,80 @@ const EditBridgeModal: React.FC<EditBridgeModalProps> = ({
   }
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col px-40">
-      <div className="bridge-edit mb-4 flex w-1/2 flex-col justify-start">
-        {editBridgeLabels.map((item, index) => (
-          <div className="mt-4 flex justify-between" key={item.key}>
-            <div
-              className={
-                index === 0
-                  ? 'flex flex-col space-y-2'
-                  : `flex flex-row space-x-2`
-              }
-            >
-              <label
-                className="min-w-[76px] bg-transparent text-lg text-[#7E7B94]"
-                htmlFor={item.key}
-              >
-                {item.value} :
-              </label>
-              <input
-                id={item.key}
-                className={`border-none bg-transparent p-0 uppercase text-white ${
-                  editModes[item.key] ? 'border-b-2 border-emerald-500' : ''
-                }`}
-                type="text"
-                key={item.key}
-                defaultValue={
-                  selectedBridge[item.key as keyof typeof selectedBridge] || ''
+    <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col px-20 md:px-40">
+      <div className="flex justify-between flex-col lg:flex-row  lg:space-x-10 mb-4">
+        <div className="bridge-edit mb-4 flex w-1/2 flex-col justify-start">
+          {editBridgeLabels.map((item, index) => (
+            <div className="mt-4 flex justify-between" key={item.key}>
+              <div
+                className={
+                  index === 0
+                    ? 'flex flex-col space-y-2'
+                    : `flex flex-row space-x-2`
                 }
-                // onChange={(e) => {}}
-                readOnly={!editModes[item.key]}
-                {...register(item.key, { required: true })}
-              />
-            </div>
-            {index !== 0 && (
-              <button
-                type="button"
-                className="text-white"
-                onClick={() => toggleEditMode(item.key)}
               >
-                <svg
-                  xmlns="http://www.w3.org/2000/svg"
-                  fill="none"
-                  viewBox="0 0 24 24"
-                  strokeWidth={1.5}
-                  stroke="currentColor"
-                  className="h-6 w-6"
+                <label
+                  className="min-w-[76px] bg-transparent text-lg text-[#7E7B94]"
```

Comment:
Could u use tailwind numbers instead of [76px]?

Reply:
fixed.